### ISSUE TITLE
Current code does not support pseudo label generation for target training set

### ISSUE THREAD

**Issue by chester256** [valeoai/xmuda#7]
When set the mode as "train", the "output_orig" will be false, while validate.py requires output_orig.
https://github.com/valeoai/xmuda/blob/8b39f8f77f048cd4c086323dd89fbf9710335d35/xmuda/test.py#L75
https://github.com/valeoai/xmuda/blob/8b39f8f77f048cd4c086323dd89fbf9710335d35/xmuda/data/build.py#L44

**Comment by maxjaritz** [comment]
The train or test mode is set independently from the dataset. You can have mode='test' and validate on the target training set.

In the README.md, I give the following example:`
```
python xmuda/test.py --cfg=configs/nuscenes/usa_singapore/xmuda.yaml --pselab @/model_2d_100000.pth @/model_3d_100000.pth DATASET_TARGET.TEST "('train_singapore',)"
```

As you use the test script `test.py`, the dataloader is in test mode (output_orig = true), but you load the `train_singapore` (target) set and generate the pseudo labels.
